htmd/parameterization/dihedral.py

```diff
@@ -410,6 +410,14 @@ def plotDihedralEnergies(self, idihed):
         fitted_energy = self._fitted_energies[idihed] - np.min(self._fitted_energies[idihed])
         indices = np.argsort(angle)
 
+        path =  os.path.join(self.result_directory, self._names[idihed])
+
+        if write_data:
+            header = 'angle  QM_ref   MM_init  MM_fit'
```

Comment:
Since we are at it, you could make the header follow the `fmt` of the `savetxt` (fixing the size in general, and changing to f or n, depending on the writing)

Reply:
Don't get your idea?

Reply:
just did a commit with my idea